packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.js

```diff
@@ -88,13 +88,11 @@ class BpkScrollableCalendarGridList extends React.Component {
     );
   }
 
-  renderList({ width, height }) {
+  renderList(width, height) {
     return (
       <List
         extraData={this.props}
-        style={
-          this.getHtmlElement().dir === 'rtl' ? { direction: 'rtl' } : null
-        }
+        style={this.getHtmlElement().dir === 'rtl' ? { direction: 'rtl' } : {}}
```

Comment:
IMO `null` is a more appropriate value than and empty object as it signals that there is no style whereas an empty object is ambiguous. Any particular reason to make this change?

Reply:
We get warnings in the console if we pass a null. It's declared in react-virtualized as `PropTypes.object.isRequired` with a default of `{}`.

Reply:
Ok doke - nice one 👍